module3/projects/take_home/rubric.md

```diff
@@ -0,0 +1,160 @@
+- asks questions to clarify the scope of the problem or states their assumptions without being prompted. 
+- is able to modify their solution independently if an edge case is pointed out.
+- builds a solution that meets all MVP expectations provided by the challenge and is easy to use by the end user.
+
+#### Leaning Don’t Hire
+The candidate...
+- is unable to arrive at a solution beyond the most basic approach. The final solution does not account for edge cases. 
+- dives in to a solution without giving the problem much thought. 
+- is only able to solve a bug or for an identified issue with significant help.
+- built most of the MVP, but one or two requirements may be incomplete.
+
+#### Strong Don’t Hire
+The candidate...
+- is unable to arrive at a solution beyond the most basic approach. 
+- fails to complete three or more requirements of the MVP or the application is broken.
+</section>
+
+<section class="dropdown">
+### 2. Technical Fluency and Code Design
+#### Strong Yes Hire
+
+The candidate...
+- chooses to use standard library functions and is able to describe their behavior when asked. 
+- is able to break down a complex system into elegantly structured components, and thoroughly describe the interaction model, the interface and the behavior Abstractions are clear and clean, given the problem definition, and keep open the possibility of future needs without significant refactoring.
+- understands time and space complexity (even if they don’t use those terms) and can describe the characteristics of their implemented code if appropriate.
+
+#### Leaning Yes Hire
+The candidate...
+- can describe the tradeoffs inherent in different approaches, and can articulate why a specific data structure is appropriate. 
+- chooses to use standard library functions and is able to describe their behavior when asked (ex: chooses ActiveRecord over plain Ruby). 
+- is able to structure their solution in a way that separates classes appropriately and demonstrates a good understanding of OOP principles (even if they don’t use the specific terms). 
+
+#### Leaning Don’t Hire
+
+The candidate...
+- is able to write basic code, but does not demonstrate strong familiarity with their chosen language’s properties or built in features. 
+- re-implements standard library functions, rather than using what is available. 
+- provides a finished output that has the appearance of being bolted together, rather than being coherently designed. 
+- builds abstractions, but are rigid or resistant to change.
+
+#### Strong Don’t Hire
+
+The candidate...
+- is familiar with common data structures, but cannot describe which ones are appropriate for the task at hand. 
+- is unable to translate their thoughts into code. 
+- is unable to speak to code or describe what it does. 
+- does not seek to break out their code into reusable components, and doesn’t understand the value of doing so.
+</section>
+
+<section class="dropdown">
+### 3. Testing and Debugging
```

Comment:
Is this whole section referring to BE testing? They will have learned FE testing by this point, so we'll want to clarify. The points are still relevant for both, but students might wonder if we expect these things for both or just one.

Reply:
I was thinking testing as a whole, both FE and BE